File: app/src/main/java/com/rokudo/xpense/utils/GsonHelper.java
package com.rokudo.xpense.utils;

import com.google.gson.Gson;
import com.rokudo.xpense.data.retrofit.models.Balances;

public class GsonHelper {
    public static String serializeBalancesToJson(Balances bmp) {
        Gson gson = new Gson();
        return gson.toJson(bmp);
    }

    // Deserialize to single object.
    public static Balances deserializeBalancesFromJson(String jsonString) {
        Gson gson = new Gson();
        return gson.fromJson(jsonString, Balances.class);
    }
}

File: app/src/main/java/com/rokudo/xpense/utils/GsonHelper.kt
package com.rokudo.xpense.utils

import com.google.gson.Gson
import com.rokudo.xpense.data.retrofit.models.Balances

object GsonHelper {
    fun serializeBalancesToJson(balances: Balances?): String = Gson().toJson(balances)

    fun deserializeBalancesFromJson(jsonString: String?): Balances? {
        return if (jsonString != null) Gson().fromJson(jsonString, Balances::class.java) else null
    }
}

